packages/dd-trace/src/debugger/devtools_client/snapshot/processor.js

```diff
@@ -0,0 +1,241 @@
+'use strict'
+
+module.exports = {
+  processRawState: processProperties
+}
+
+// Matches classes in source code, no matter how it's written:
+// - Named:                          class MyClass {}
+// - Anonymous:                      class {}
+// - Named, with odd whitespace:     class\n\t MyClass\n{}
+// - Anonymous, with odd whitespace: class\n{}
+const CLASS_REGEX = /^class\s([^{]*)/
+
+function processProperties (props, maxLength) {
+  const result = {}
+
+  for (const prop of props) {
+    // TODO: Hack to avoid periods in keys, as EVP doesn't support that. A better solution can be implemented later
+    result[prop.name.replaceAll('.', '_')] = getPropertyValue(prop, maxLength)
+  }
+
+  return result
+}
+
+function getPropertyValue (prop, maxLength) {
```

Comment:
Yeah, I wrote this knowing it would have to be refactored once we introduced the other limits in an upcoming PR. But I thought it would make this PR simpler to not consider that for now. Hence the current implementation :)